docs-2.0/nebula-dashboard-ent/2.deploy-connect-dashboard-ent.md

```diff
@@ -6,11 +6,7 @@ This topic will introduce how to install and deploy Dashboard in detail.
 
 Before deploying Dashboard, you must do a check of these:
 
-- Select and download Dashboard of the correct version. The version correspondence between Dashboard and Nebula Graph is as follows.
-
-  | Dashboard version           | Nebula Graph version |
-  | :-------------------------- | :--------------- |
-  | {{ dashboard_ent.release }} | 2.x              |
+- Select and download Dashboard of the correct version. For information about the version correspondence between Dashboard and Nebula Graph, see [Version compatibility](1.what-is-dashboard-ent.md).
```

Comment:
Why not use the link of the Version compatibility section

Reply:
> Why not use the link of the Version compatibility section

Orz, Copper said it's easy to lead to errors, we'd better avoid quoting sections.